extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -195,7 +197,8 @@
 
 /**
  * Base test runner for running MSQ unit tests. It sets up multi stage query execution environment
- * and populates data for the datasources. The runner does not go via the HTTP layer for communication between the
+ * and populates data for the datasources. The ruimport org.apache.druid.initialization.CoreInjectorBuilder;
+nner does not go via the HTTP layer for communication between the
```

Comment:
"ruimport" rogue copy/paste?

Reply:
Misclick and dragged and import in the comment. Thanks for pointing it out.